activity_heartbeating/worker.rb

```diff
@@ -4,14 +4,16 @@
 require_relative 'my_workflow'
 require 'logger'
 require 'temporalio/client'
+require 'temporalio/env_config'
 require 'temporalio/worker'
 
+# Load config and apply defaults
+positional_args, keyword_args = Temporalio::EnvConfig::ClientConfig.load_client_connect_options
+positional_args = ['localhost:7233', 'default'] if positional_args.empty?
+keyword_args[:logger] = Logger.new($stdout, level: Logger::INFO)
```

Comment:
Would not recommend mutating this, but rather passing `logger: ` to the `connect` call itself before splatting the kwargs. Same later for `interceptors` and any others.